frontend/src/utilities/useWatchBuildStatus.tsx

```diff
@@ -24,7 +24,7 @@ export const useWatchBuildStatus = (): BuildStatus[] => {
   const dispatch = useAppDispatch();
 
   React.useEffect(() => {
-    let watchHandle: ReturnType<typeof setTimeout>;
+    let watchHandle: ReturnType<typeof setTimeout> | undefined;
```

Comment:
Remove the if condition in the return statement.
```suggestion
    let watchHandle: ReturnType<typeof setTimeout>;
```